config/db/easyfood_database.sql

```diff
@@ -83,3 +84,20 @@ INSERT INTO `usuarios` (`primeiro_nome`, `ultimo_nome`, `telefone`, `email`, `se
   ('Joao', 'Pedro', '31984464729', 'joaopedro@gmail.com', SHA1('joaopedro2010')),
   ('Maria', 'Lima', '35987432164', 'marialima@hotmail.com', SHA1('ml15122015')),
   ('Carlos', 'Antunes', '37984455792', 'carlos_antunes12@outlook.com', SHA1('carlitos1212'));
+
+									    
+-- Dumping structure for table easyfood.reviews
+CREATE TABLE IF NOT EXISTS `reviews` (
+  `id` INT(11) unsigned NOT NULL AUTO_INCREMENT,
+  `id_usuario` INT(11) unsigned NOT NULL DEFAULT 0,
+  `id_restaurante` INT(11) unsigned NOT NULL DEFAULT 0,
+  `nota` tinyint(1) unsigned NOT NULL DEFAULT 0,
+  `comentario` varchar(600) NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `id_usuario` (`id_usuario`),
+  KEY `id_restaurante` (`id_restaurante`),
+  CONSTRAINT `id_usuario` FOREIGN KEY (`id_usuario`) REFERENCES `usuarios` (`id`) ON DELETE SET NULL ON UPDATE CASCADE,
```

Comment:
A declaração id_usuario está como NOT NULL na tabela, porém o tratamento de ON DELETE da constraint é SET NULL, o que irá gerar conflito